backends-velox/src/test/scala/io/glutenproject/execution/VeloxWholeStageTransformerSuite.scala

```diff
@@ -114,7 +114,11 @@ class VeloxWholeStageTransformerSuite extends WholeStageTransformerSuite {
       val wholeStages = plan.collect {
         case wst: WholeStageTransformerExec => wst
       }
-      assert(wholeStages.length == 1)
+      if (SparkShimLoader.getSparkVersion.startsWith("3.3.")) {
+        assert(wholeStages.length == 3)
```

Comment:
Can we use the below code instead?
```
if (VersionUtils.majorVersion(SparkShimLoader.getSparkVersion) == 3 && VersionUtils.minorVersion(SparkShimLoader.getSparkVersion) == 2) {
 assert(wholeStages.length == 1)
} else {
 assert(wholeStages.length == 3)
}
```
Thus, we can avoid to change the condition after we upgrading to higher spark version, e.g., future 3.4.x, assuming it is still applicable to such higher versions.